src/scripts/main.js

```diff
@@ -1 +1,74 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.documentElement.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  // eslint-disable-next-line prefer-promise-reject-errors
+  window.setTimeout(() => reject('First promise was rejected'), 3000);
```

Comment:
The first promise has a rejection handler that triggers after 3000ms if the promise is not resolved. Ensure this behavior is intended, as it may lead to unexpected rejections if the user does not click within this timeframe.